hieroplatform/src/test/java/org/hiero/sketch/TopKSketchTest.java

```diff
@@ -0,0 +1,37 @@
+package org.hiero.sketch;
+
+import org.hiero.sketch.spreadsheet.ColumnSortOrientation;
+import org.hiero.sketch.spreadsheet.NextKList;
+import org.hiero.sketch.spreadsheet.TopKSketch;
+import org.hiero.sketch.table.RecordOrder;
+import org.hiero.sketch.table.Table;
+import org.junit.Test;
+
+import static org.hiero.sketch.TableTest.getRepIntTable;
+
+public class TopKSketchTest {
+
+
+    @Test
+    public void testTopK() {
+        final int numCols = 2;
+        final int maxSize = 50;
+        final int rightSize = 1000;
+        final int leftSize = 1000;
+        final Table leftTable = getRepIntTable(leftSize, numCols);
+        //System.out.println(leftTable.toLongString(50));
```

Comment:
Please remove the println, and replace them with asserts.
Maybe you want to use toString instead for the asserts.